### PR TITLE
Service UI

### LINKED ISSUE #13: Service page area
This is where all services provided by this app are displayed

### PR DESCRIPTION
**Pull request by ekiira**
## Description
This is where services provided by this solution or product are described. It consist of service one can get from using the product.

Fixes #13 #4

How Has This Been Tested?
This is where services provided by this solution or product are described. It consist of service one can get from using the product.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [x] I have added tests that prove my fix is effective and that this feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Taiwrash — DISMISSED** [review]
Add issue #4 to your PR to make it two issues. Thanks well done job

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_CLOUDINARY_URL ='https://api.cloudinary.com/v1_1/ekiira/image/upload'
+REACT_APP_CLOUDINARY_UPLOAD_PRESET ='jbk6xkqw'
```
Comment (triple0t) [inline .env]: add your .env to your gitignore file

File: public/css/style.css [hunk 2]
```diff
@@ -412,8 +449,8 @@ h6 {
 }
 
 #hero .btn-get-started:hover {
-  background: #084933;
-  border: 2px solid #084933;
+  background: #0f2719;
+  border: 2px solid #0f2719;
 }
 
 @media (min-width: 1024px) {
```
Comment (triple0t) [inline public/css/style.css]: I can't see a lot of Mobile-specific styling here
Is Mobile Responsiveness been considered?
Also, add screen to your media query

File: src/App.test.js [hunk 3]
```diff
@@ -1,9 +1,105 @@
...
+    );
+  });
+
+  it('should show text', () => {
+    const home = shallow(<Home />);
+    const content = home.find("[to='/accounts']");
+    expect(content.text()).toBe('Get Started');
+  });
+
+  it('should show text', () => {
```
Comment (triple0t) [inline src/App.test.js:168]: You are currently running UI testing, not functionality testing
This is fine for now but remember to update your test cases to test for features and functionality as well

**Review by triple0t — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (ekiira) [reply .env]: Thanks I will

File: public/css/style.css (hunk 2 above)
Comment (ekiira) [reply public/css/style.css]: Okay, And Yes It is mobile friendly

File: src/App.test.js (hunk 3 above)
Comment (ekiira) [reply src/App.test.js:168]: Alright Thanks

**Review by triple0t — DISMISSED** [review]

**Review by Taiwrash — DISMISSED** [review]

**Pull request merged**